webpack.common.config.js

```diff
@@ -1,15 +1,43 @@
 const path = require('path');
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
 
 module.exports = {
   devServer: {
     https: true
   },
-
+  resolve: {
+    extensions: [
+      '.ts',
+      '.tsx',
+      '.js',
+      '.json',
+      '.jsx',
```

Comment:
Do we still have any `*.jsx` files to handle? If not, I suggest to remove support for these files completely.